### ISSUE TITLE
Add a button to reset car mods to stock in Vehicle Mods

### ISSUE THREAD

**Issue by saldin93** [itsjustcurtis/MenyooSP#178]
Hi there, 

Since we have a button to 'max upgrade' a car, hoping we could get a button to undo/reset a car to stock w/o spawning another car.

Thank you!

**Comment by TayMcKenzieNZ** [comment]
That's actually a really good suggestion! Hoping he considers it 👀😏😎

**Comment by itsjustcurtis** [comment]
Agreed; shouldn't be too hard to implement so I'll knock this one out for the next patch

**Comment by itsjustcurtis** [comment]
Apparently this took 2 years, but it was added with the most recent update.
